asv_bench/benchmarks/generic_dtypes.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pandas as pd
+
+from pandas.core.dtypes import generic
+
+abcs = [getattr(generic, name) for name in dir(generic)
+        if name.startswith('ABC')]
+
+
+class GenericDtypes(object):
+    def setup(self):
```

Comment:
you need a goal time

you need separate asv for each isinstance there is a way of doing this generically so it matrixes the tests
look in the other benches i think for bottleneck

Reply:
What constitutes a reasonable goal time?

Reply:
i think if it's set too high it doesn't show the significantly changes properly 
have a look in asv docs as to actual purpose

Reply:
Looks like this param adjusts the number of iterations to target the total runtime.  Default is 0.1